test/upgrade/prober/verify.go

```diff
@@ -37,23 +37,29 @@ import (
 const (
 	fetcherName     = "wathola-fetcher"
 	jobWaitInterval = time.Second
-	jobWaitTimeout  = 5 * time.Minute
+	jobWaitTimeout  = 1 * time.Minute
 )
 
 // Verify will verify prober state after finished has been sent.
 func (p *prober) Verify() (eventErrs []error, eventsSent int) {
-	report := p.fetchReport()
+	var report *receiver.Report
+	p.log.Info("Waiting for complete report from receiver...")
+	start := time.Now()
+	if err := wait.PollImmediate(time.Second, 5*jobWaitTimeout, func() (bool, error) {
```

Comment:
This pooling config could be adjusted a bit. I'd say waiting just a second between deploying fetchers jobs seams like a small amount.

```suggestion
	if err := wait.PollImmediate(jobWaitInterval, jobWaitTimeout, func() (bool, error) {
```

Reply:
A few notes on this:
* The run of each fetcher job takes several seconds, the next attempt is only done once the previous fetch ends - so overall the interval is not 1 second but it's about 3-5 seconds. using `jobWaitInterval` would increase this to something like 10 seconds? I've chosen the smallest reasonable interval here because I actually want to run the next round more or less immediately
* I don't want to use the constant `jobWaitInterval`, this constant is used somewhere else for a different purpose - waiting for the job to succeed. The fact that this const has the same value is just a coincidence.
* jobWaitTimeout is already used when waiting for the single job to succeed. Having the same value here doesn't make sense because when the other job takes so much time and times out then this timeout will have already passed so there won't be another attempt. That's why I've chosen it to be a multiple of the other timeout.
I'd also say that having just one minute for the timeout is not enough. It's too limiting

Reply:
Hmm, maybe I'd just use this:
```
if err := wait.PollImmediate(time.Second, jobWaitTimeout, func() (bool, error) {
```
and keep the jobWaitTimeout at 5 minutes

Reply:
:+1: